src/lib/dialog/dialog.scss

```diff
@@ -0,0 +1,52 @@
+@import '../core/style/elevation';
+@import '../a11y/_a11y';
+
+
+$md-dialog-padding: 24px !default;
+$md-dialog-border-radius: 2px !default;
+$md-dialog-max-width: 80vw !default;
+$md-dialog-max-height: 65vh !default;
```

Comment:
actually yes, there are fullscreen dialogs in material guidelines https://material.io/guidelines/components/dialogs.html#dialogs-confirmation-dialogs